sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClientTest.java

```diff
@@ -1493,7 +1493,7 @@ public void testFormulaPrebuiltRead(HttpClient httpClient,
             DocumentFormula formula =
                 analyzeResult.getPages().get(0).getFormulas().get(0);
             Assertions.assertEquals(DocumentFormulaKind.INLINE, formula.getKind());
-            Assertions.assertEquals("a + b = c .", formula.getValue());
+            Assertions.assertTrue(formula.getValue().startsWith("a + b ="));
```

Comment:
Just a suggestion can you just check that the string is not empty instead of checking for the value?